LoRaEngine/modules/LoRaWanPktFwdModule/Dockerfile.arm32v7

```diff
@@ -3,16 +3,18 @@ FROM sdthirlwall/raspberry-pi-cross-compiler as builder
 RUN apt-get install -y git
 RUN git clone https://github.com/Lora-net/packet_forwarder.git
 RUN git clone https://github.com/Lora-net/lora_gateway.git
+# For RAK833 based modules
+RUN sed -i "s|# write output for SX1301 reset|echo \"0\" > /sys/class/gpio/gpio\$IOT_SK_SX1301_RESET_PIN/value; WAIT_GPIO|g" ./lora_gateway/reset_lgw.sh
 WORKDIR /build/packet_forwarder
 RUN ./compile.sh
 WORKDIR /build
 
 
-FROM microsoft/dotnet:2.0-runtime-stretch-arm32v7 as exec
+FROM arm32v7/debian:stretch-slim as exec
 WORKDIR /LoRa
 COPY --from=builder /build/packet_forwarder/lora_pkt_fwd/lora_pkt_fwd .
 COPY --from=builder /build/packet_forwarder/lora_pkt_fwd/global_conf.json .
 COPY --from=builder /build/lora_gateway/reset_lgw.sh .
 COPY local_conf.json .
 COPY start_pktfwd.sh .
-ENTRYPOINT ["./start_pktfwd.sh","7"]
```

Comment:
Is 7 the default value here?

Reply:
there is no more default value we need to pass it as env variable and we need to update the arm template to reflect this change too. The intel and RAK833 uses 5, seeedstudio 7, iC880A 13 (still need to be tested)

Reply:
My RAK833 actually used 12. I change the defaults in the template to 7 for ARM (seedstudio) and 5 for X86_64 .